src/Service.Tests/SqlTests/GraphQLQueryTests/GraphQLQueryTestBase.cs

```diff
@@ -1573,7 +1573,7 @@ public virtual async Task TestInvalidFirstParamQuery()
         {
             string graphQLQueryName = "books";
             string graphQLQuery = @"{
-                books(first: -1) {
+                books(first: -2) {
```

Comment:
add test comment that describes that this test ensures `first` value of <-1 results in request failure. may also need to add assert on substatus code if you emit a substatus code aligning with bad `first` param.

Reply:
fixed this.